salt/kubernetes-common/openstack-config.jinja

```diff
@@ -0,0 +1,16 @@
+[Global]
+auth-url={{ pillar['cloud']['config']['auth_url'] }}
+username={{ pillar['cloud']['config']['username'] }}
+password={{ pillar['cloud']['config']['password'] }}
+tenant-name={{ pillar['cloud']['config']['project'] }}
+domain-name={{ pillar['cloud']['config']['domain'] }}
+region={{ pillar['cloud']['config']['region'] }}
+[LoadBalancer]
+lb-version=v2
+subnet-id={{ pillar['cloud']['config']['subnet'] }}
```

Comment:
And, after a bit more digging, there is a few patches that made it to Stackato (and hpcloud/kubernetes) that did not make it upstream. (or at least are not there now). We should hold off on the security groups.